docs/api/toggle.md

```diff
@@ -60,6 +87,28 @@ import CSSParts from '@site/static/usage/v7/toggle/theming/css-shadow-parts/inde
 
 <CSSParts />
 
+## Migrating from Legacy Toggle Syntax
+
+A simpler toggle syntax was introduced in Ionic 7.0. This new syntax reduces the boilerplate required to setup an toggle, resolves accessibility issues, and improves the developer experience.
+
+While developers can continue using the legacy syntax, we recommend migrating as soon as possible.
+
+### Using the Modern Syntax
```

Comment:
(Putting this here for threading purposes) I noticed that when opening the playgrounds in Stackblitz with React specifically, the toggles appear in the middle of the page. If there are multiple toggles with labels (such as when adding the `<>` tags manually to the new playgrounds that need them), each toggle seems to take up the whole height of the page, leading to a lot of white space. 🤔

Reply:
Good catch! This is a bug. Opened a PR at https://github.com/ionic-team/ionic-framework/pull/26611 to fix

Reply:
Once the fix is merged I'll update the release-7.0 branch with a new dev build

Reply:
Updated in https://github.com/ionic-team/ionic-docs/pull/2706